static/js/components/UserManagement.jsx

```diff
@@ -132,14 +131,7 @@ const UserManagement = () => {
     !acls?.length ||
     !roles?.length
   ) {
-    return (
-      <Box
-        display={queryInProgress ? "block" : "none"}
```

Comment:
Double checking that you meant to remove the conditional rendering here.  Otherwise looks good.

Reply:
yeah I did, I think that conditional rendering stops the loading icon from appearing on initial page load.

Reply:
Great, thanks for checking.